shared/js/background/tab-manager.es6.js

```diff
@@ -1,22 +1,67 @@
 const Companies = require('./companies.es6')
 const settings = require('./settings.es6')
 const Tab = require('./classes/tab.es6')
+const utils = require('./utils.es6')
+const browserWrapper = require('./$BROWSER-wrapper.es6')
+let browser = utils.parseUserAgentString()
 
 class TabManager {
     constructor() {
         this.tabContainer = {}
     };
 
+    /* Get stashed tabId from native safari tabs. This needs to 
+     * be here for now. The popup can't access the tabId for
+     * some reason
+     */
+    getTabId(e) {
```

Comment:
These will need to stay here for now. There's some bug with accessing the active safari tab from inside the popup that makes no sense to me.